src/routes/get-started/developers/upgrade-guide/ReactGuide.tsx

```diff
@@ -0,0 +1,91 @@
+import { CodeSnippet } from "@components/code-snippet/CodeSnippet.tsx";
+import { GoATable } from "@abgov/react-components";
+import { InlineCode } from "@components/inline-code/InlineCode.tsx";
+import { components } from "@routes/get-started/developers/upgrade-guide/components.ts";
+
+export const ReactGuide = () => {
+  const componentNames = components.filter(x => x.react);
+  return (
+    <>
+      <h2 id="react">Migrating a React app</h2>
+      <p>
+        The React components for the DDD Design System are still located in the{" "}
+        <InlineCode>@abgov/react-components</InlineCode> package. However, starting with version 6,
+        component properties should be imported from{" "}
+        <InlineCode>@abgov/ui-components-common</InlineCode> instead of{" "}
+        <InlineCode>@abgov/react-components</InlineCode>.
+      </p>
+
+      <h4>1. Update dependencies</h4>
+      <CodeSnippet
+        lang="typescript"
+        allowCopy={true}
+        code={`
+          npm i @abgov/web-components@latest
+          npm i @abgov/react-components@latest
+          npm i @abgov/ui-components-common@latest
```

Comment:
The `ui-components-common` is being moved into the React lib, so it will no longer be needed once that is done